.github/workflows/performance.yml

```diff
@@ -178,74 +199,79 @@ jobs:
       - name: Run performance tests (current commit)
         run: npm run test:performance
```

Comment:
The playwright downloads extensive zips for both Firefox and Webkit. Do we need that? Is there any way we can exclude it to speed up our workflow execution?

Reply:
Maybe, I‘d have to try it